src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -125,6 +125,7 @@ const TransactionDetailsComponent = (props: Props) => {
         returnKeyType="done"
         keyboardType="numeric"
         submitLabel={lstrings.string_save}
+        textSizeRem={1.5}
```

Comment:
What makes this field deserve a different text size? I don't think our input modals should even have a prop like this to begin with, we should use one consistent size or else this just looks messy.

Reply:
That's what we use in the FiatAmount scene so it matches. The text field looks bizarringly small without this change

Reply:
![image](https://github.com/EdgeApp/edge-react-gui/assets/6079354/716d10ad-f0a6-4d22-bbf4-9fd5df90a702)

<img width="466" alt="Screenshot 2024-03-09 at 18 49 27" src="https://github.com/EdgeApp/edge-react-gui/assets/6079354/bbe28e13-9a98-49d3-bf8d-e77bba80a72a">

Reply:
To me, `FiatPluginEnterAmount` scene seems weird since it's inconsistent with our app for various things like spacing and field width/height. I don't think FPEA should be the basis that the rest of the app adheres to, but rather the other way around. If we have reasons for it to be different, I think it needs to be appear _obviously_ different. Right now I think it looks like unintentional inconsistency since it's so slightly different. Also, this modal isn't even seen from the `FiatPluginEnterAmount` scene so there's no way for the user to even make that association.

In a vacuum, though, if input modals in general are too small, I agree with the change on that basis if we are also comfortable changing ALL text modals, and to remove this prop. Consistency I think is less debatable between like components.